storage/sequence_cache.go

```diff
@@ -104,6 +104,33 @@ func (sc *SequenceCache) Get(e engine.Engine, id []byte, dest *roachpb.SequenceC
 	return epoch, seq, nil
 }
 
+// GetAllID returns all the key-value pairs for the given ID from the engine.
```

Comment:
s/GetAllID/GetAllTransaction/ (and s/ID/transaction ID/ throughout the file)

The interfaces of all of these methods are somewhat inconsistent (`Get` and `Iterate` decode the value proto, but this method does not; `Get` and `Put` treat the storage as a black box but the two new methods expose the underlying keys directly). But this is also exactly what the callers need so I'm not sure if there are any worthwhile changes to be made.